mobile/src/filtersModal.js

```diff
@@ -40,12 +43,20 @@ export default class FilterSelect extends Component {
     return (
       <View style={s.buttonContainer}>
         <TouchableOpacity onPress={() => this.props.handleChange("showFilterSelect", false)}><Text style={s.closeButton}>X</Text></TouchableOpacity>
-        <Text style={s.title}>Sort</Text>
+        <Text style={s.title}>Lists</Text>
         <Text style={{width: 25}}></Text>
       </View>
     )
   }
 
+  handleChange = (item) => {
```

Comment:
Let's name this function to reflect what is changing.  Prefixing with `on` is idiomatic, e.g. `onSortChange`